packages/web-runtime/src/pages/logout.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div>
+    <p>Please wait while you're being logged out.</p>
```

Comment:
```suggestion
    <p v-translate>Please wait while you're being logged out.</p>
```

Please add the `v-translate` directive, so that the english string gets picked up by our extraction scripts for translation.